package.json

```diff
@@ -19,13 +19,17 @@
   "license": "ISC",
   "author": "Bruno Barbieri",
   "main": "index.js",
+  "types": "@types/index.d.ts",
   "files": [
-    "index.js"
+    "index.js",
+    "trezor-keyring.js",
+    "trezor-keyring-mv2.js",
+    "@types/index.d.ts"
   ],
   "scripts": {
     "setup": "yarn install && yarn allow-scripts",
     "test": "mocha",
-    "lint:eslint": "eslint . --cache --ext js,ts",
+    "lint:eslint": "eslint . --cache --ext js,ts --ignore-pattern @types/index.d.ts",
```

Comment:
I don't like the idea of ignoring it, but given that we don't have typescript support here eslint was being a bit of a hassle.

Reply:
Maybe this can be added to `.eslintrc.js` instead?